plugins/scenario/README.md

```diff
@@ -2,15 +2,19 @@
 
 
 The Gamma Scenario Language is based on the Live Sequence Charts formalism and allows the definition of scenarios that describe executions of Gamma systems. Scenarios contain interactions sent or received by the component, delays and combined fragments.
-Interactions can either be cold or hot, indicating, weather the given interaction is optional or mandatory for the execution to be accepted. To extend the expressive power of the language, annotations can be used. 
+Interactions can either be cold or hot, indicating, weather the given interaction is optional or mandatory for the execution to be accepted. To extend the expressive power of the language, annotations can be used.
+Furthermore, variables can be declared in scenarios. Check expressions can be used to define constraints for the expected behaviour. 
 
 Based on the scenarios tests can be generated, which can verify that the implementation of the system holds up to the expected behaviour. 
 
 A more complete description can be found in the docs sub folder in the Scenario Tutorial document.
 
+For further verification of the system, negative tests can also be derived from scenarios. 
+
+Furthermore, the framework allows the generation of scenario-based monitor automata. These automata can be used for runtime verification.
+
 Known problems:
 
-1. A large number of abstract tests are generated based on scenarios.
-2. The Theta framework, which is used as a formal verification backend, does not produce all possible paths to the Accepting state in all scenarios.
-3. If an interaction set contains a delay and atomic interactions, the timer of the delay starts again in every iteration of the interval. 
-4. Currently it is not possible, to describe, that an interaction should be sent immediately after the start of the scenario as at the start of the formal verification process, no events are present in the system.
\ No newline at end of file
+1. Check expressions can set an acceptable interval for an event parameters value. Currently each possible value from an interval leads to a separate test. A possible method to mitigate this problem could be to allow the user choose an option where only a select few tests are produced. For example, 5 tests could cover the middle of the interval, the lover and upper bounds and cases just above and below the bounds.
+2. During test generation, negated interactions are not back-annotated to abstract tests.
+3. Waiting directly before combined fragments can only described by using severaly delay expressions. A possible solution would be to allow the description of such structures: _{delay alternative{}or{}}_. Currently this is not allowed in the scenario language, due to the complexity it would introduce to the scenario-based transformations.
```

Comment:
"Waiting (i..e., delays) directly before combined fragments can be described only"